python/simulation.py

```diff
@@ -131,7 +131,7 @@ def __init__(self, center, size=Vector3(), dims=2,):
         self.swigobj = mp.volume(vec1, vec2)
 
 
-Mode = namedtuple('Mode', ['freq', 'Q', 'amp', 'err'])
+Mode = namedtuple('Mode', ['freq', 'freq_imag', 'Q', 'amp', 'err'])
```

Comment:
`freq_imag` could be renamed `decay`, since it is called the decay rate in the harminv documentation.

Reply:
Done